md/sap-jco-3.md

```diff
@@ -32,38 +32,44 @@ JCo 3: `sapjco3-ntamd64-3.0.3`
-3. Put the `sapjco3.jar` file in the shared libraries directory of Bonita BPM, so that the jar is in the classloader used by the Bonita BPM Engine.
-4. Put the native `sapjco.dll` or `.so` libraries in the java library path or the system path.
-
-### How to import the SAP JCo3 library and make a request using an example function using the graphic display
-
-1. Click _**Import SAPjco3.jar**_, and use the browse dialog box to locate and select one of the .jar file
-2. Click _**Open**_ to import the .jar file contents
-3. Click _**OK**_ to reboot the machine and save the configuration.
-4. When the connector window is displayed, the import .jar file button will have disappeared. This is replaced by **disabled** button. A .jar file cannot be imported twice.
-5. Fill in all the connection fields with the correct settings. Note that the SAP JCo 3 connector supports a single destination for all connector instances in Bonita BPM Engine.
-6. Once all the fields are filled, test the connection by clicking on the _**test **_button.
-7. In the function definition window, only the function name is required.
-8. Click on the arrow at the end of the field to display a dropdown list of functions.
-9. Click on `GET_SYSTEM_NAME` for example
-10. Click _**Next**_
-11. This will display the **Input Parameters** window
-12. Click _**Next**_
-13. This will display the **Output Parameters** window
-14. The `GET_SYSTEM_NAME` is displayed to confirm the input is this function. Note: You can create a blank .html file on your c: drive, which will be filled with the output information from the SAP connector
-15. Enter the path to the .html file in the field called "HTML File"
+3. Put the `sapjco3.jar` file in the webapp libraries directory of the application server, so that the jar is in the classloader used by the Bonita BPM Engine.
+4. Put the `sapjco.dll` or `.so` libraries in the native library search path: `C:\windows\system32` for windows, or one of the directory listed in the `LD_LIBRRARY_PATH` for Linux/UNIX.
+
+### Studio: How to import the SAP JCo3 library and make a request using an example function using the graphic display
+
+This is a Windows example and it is assumed that the Studio had been started at least once successfully.
+
+1. Store the `sapjco.dll` in the `C:\windows\system32` directory
+2. Reboot
+3. Store the `sapjco3.jar` in the bonita webpapp library directory, deployed in the tomcat embedded with the Studio: e.g. `C:\BonitaBPMSubscription-7.2.3\workspace\tomcat\webapps\bonita\WEB-INF\lib`
+4. Store the `sapjco3.jar` in the `endorsed` directory of your Bonita BPM Studio installation: e.g. `C:\BonitaBPMSubscription-7.2.3\endorsed`.
+5. Start the Studio
+6. Open a diagram
+7. Select a Service task 
+8. Add a SAP JCo 3 connector
+9. Fill in all the connection fields with the correct settings. Note that the SAP JCo 3 connector supports a single destination for all connector instances in Bonita BPM Engine.
```

Comment:
Not clear here : "Note that the SAP JCo 3 connector supports a single destination for all connector instances in Bonita BPM Engine". Should it be understood as : 
"the SAP JCo 3 connector supports  _only_ a single destination for all connector instances in Bonita BPM Engine" ?

If that is the case, it might also be good to add a comment on what to do if you want more destinations.

Reply:
Well I do not know, ask the guy who wrote these lines ...